actions/hls_sponge/tests/test_0x10141001.sh

```diff
@@ -61,67 +61,46 @@ done
 
 export PATH=$PATH:${SNAP_ROOT}/software/tools:${ACTION_ROOT}/sw
 
-#### VERSION ##########################################################
-
-# [ -z "$STATE" ] && echo "Need to set STATE" && exit 1;
-
-if [ -z "$SNAP_CONFIG" ]; then
-	echo "Get CARD VERSION"
-	snap_maint -C ${snap_card} -v || exit 1;
-	snap_peek -C ${snap_card} 0x0 || exit 1;
-	snap_peek -C ${snap_card} 0x8 || exit 1;
-	echo
-fi
+snap_maint -C ${snap_card} -v || exit 1;
```

Comment:
You can add that if you made sure that software simulation works ok. So SNAP_CONFIG=CPU must work also with snap_maint ... what it currently does not.